air/src/proof/ood_frame.rs

```diff
@@ -291,37 +233,26 @@ impl<E: FieldElement> TraceOodFrame<E> {
         }
     }
 
-    /// Hashes the main, auxiliary and Lagrange kernel frame in a manner consistent with
+    /// Hashes the main, auxiliary frames in a manner consistent with
     /// [`OodFrame::set_trace_states`], with the purpose of reseeding the public coin.
     pub fn hash<H: ElementHasher<BaseField = E::BaseField>>(&self) -> H::Digest {
-        let (mut trace_states, mut lagrange_trace_states) = self.to_trace_states();
-        trace_states.append(&mut lagrange_trace_states);
+        let trace_states = self.to_trace_states();
 
         H::hash_elements(&trace_states)
     }
 
-    /// Returns the Lagrange kernel frame, if any.
-    pub fn lagrange_kernel_frame(&self) -> Option<&LagrangeKernelEvaluationFrame<E>> {
-        self.lagrange_kernel_frame.as_ref()
-    }
-
     /// Returns true if an auxiliary frame is present
     fn has_aux_frame(&self) -> bool {
         self.current_row.len() > self.main_trace_width
     }
 
-    /// Returns the main/aux frame and Lagrange kernel frame as element vectors as described in
-    /// [`OodFrame::set_trace_states`].
-    fn to_trace_states(&self) -> (Vec<E>, Vec<E>) {
+    /// Returns the main/aux frames as element vectors. Specifically, the main and auxiliary frames
+    /// are interleaved, as described in [`OodFrame::set_trace_states`].
+    fn to_trace_states(&self) -> Vec<E> {
```

Comment:
I don't think we need to change the comment here.

Reply:
Indeed, probably happened during the rebase